src/utils/helpers.js

```diff
@@ -13,27 +13,33 @@ import { useEffect, useState } from 'react';
 import { transparentize } from 'polished';
 import { CalciteTheme } from '../CalciteThemeProvider';
 
-const unitCalc = (operand1, operand2, operator) => {
-  let operand1Value = operand1;
-  let operand1Unit;
-  if (typeof operand1 === 'string') {
-    operand1Value = parseFloat(operand1);
-    operand1Unit = operand1.replace(operand1Value, '');
+const unitCalc = (left, right, operator) => {
+  let leftUnit, rightUnit;
+  if (typeof left === 'string') {
+    leftUnit = left.replace(parseFloat(left), '');
+    left = parseFloat(left);
   }
-
-  let operand2Value = operand2;
-  let operand2Unit;
-  if (typeof operand2 === 'string') {
-    operand2Value = parseFloat(operand2);
-    operand2Unit = operand2.replace(operand2Value, '');
+  if (typeof right === 'string') {
+    rightUnit = right.replace(parseFloat(right), '');
+    right = parseFloat(right);
   }
 
-  let value =
-    typeof window !== 'undefined' &&
-    window.eval(operand1Value + operator + operand2Value); // eslint-disable-line no-eval
-  value = value + (operand1Unit || operand2Unit);
+  const unit = leftUnit || rightUnit;
 
-  return value;
+  switch (operator) {
+    case '+':
+      return left + right + unit;
+    case '-':
+      return left - right + unit;
+    case '*':
+      return left * right + unit;
+    case '/':
+      return left / right + unit;
+    case '%':
+      return (left % right) + unit;
+    default:
+      return undefined;
```

Comment:
@codylawson what do you think the default case should return here? Should it `console.warn` in addition to returning something?

Reply:
Yeah I like the idea of maybe just returning `left` and a `console.warn` in the default.

Reply:
```suggestion
      console.warn('unitCalc only supports the following operators: +, -, *, /, %');
      return left;
```

Reply:
LGTM